src/commands/rover/WhoisCommand.js

```diff
@@ -31,11 +31,16 @@ class WhoisCommand extends Command {
   // This is probably the worst file in the entire project, so prepare yourself.
   // TODO: DRY this up and make the method that gets user data a method in DiscordMember
   async fn (msg, args) {
-    const member = args.member
+    let member = args.member
+    // If no member is specified the member is the message author
+    if (!member) member = msg.member
     let data = {}
     if (member) { // If the member specified exists,
       const editMessage = await msg.reply(`:mag: Looking up ${member.displayName.replace(/@/g, '')}`)
 
+      if (member.user.bot) {
+        return editMessage.edit(':robot: RoVer cannot look up bots.')
```

Comment:
Just return. no reply

Reply:
Wouldn't that leave the first message (giving the appearance the command is broken)?

Reply:
No message

Reply:
[Latest commit](https://github.com/evaera/RoVer/pull/292/commits/c0f3a8bbc1bb75a91c09be23dd662453cb3ff113) should do it